src/js/plumbing/ServerIO.js

```diff
@@ -37,7 +37,7 @@ ServerIO.PROFILER_ENDPOINT = `${C.HTTPS}://${C.SERVER_TYPE}profiler.good-loop.co
 // ServerIO.AS_ENDPOINT = `${C.HTTPS}://${C.SERVER_TYPE}as.good-loop.com`;
 // ServerIO.AS_ENDPOINT = `https://testas.good-loop.com`;
 // Use the live adserver, since our showcase ad selectionis hard-coded to live ads.
-ServerIO.AS_ENDPOINT = 'https://as.good-loop.com';
+ServerIO.AS_ENDPOINT = 'http://as.good-loop.com';
```

Comment:
Why http? Thanks. We normally want urls to be https, as some systems and browser requests demand that.

NB: Local urls are an exception 'cos local ssl is a pain to setup.